### PR TITLE
feat: adding --no_escape flag

### PR DESCRIPTION
**Pull request by lucasbazan**

### CODE REVIEW

File: telegram_notifier/__init__.py
Comment (jon-edward) [inline telegram_notifier/__init__.py:1]: Why is `from . import *` required here?

File: telegram_notifier/__main__.py
Comment (jon-edward) [inline telegram_notifier/__main__.py]: I'd prefer `"do not escape special characters (e.g. '.', '-')"` here to fit the form of the other help messages

**Review by jon-edward — CHANGES_REQUESTED** [review]
Thank you for your contributions!

I can see this being useful for people to be able to override the default escape sequence with their own behavior. 

I've made some minor suggestions but overall it looks great. I'd be ready to merge after the comments are addressed.

File: telegram_notifier/__init__.py
Comment (lucasbazan) [reply telegram_notifier/__init__.py:1]: Hello Jon. Sorry bout' that, you can delete this line after merge.
It was a test that I made lol

**Review by lucasbazan — COMMENTED** [review]
Done, I've changed the file lol

**Review by jon-edward — APPROVED** [review]
Looks good to me, thank you again for adding this.

**Comment by lucasbazan** [comment]
Thank you bro!

**Pull request merged**
